pkg/master/master.go

```diff
@@ -192,6 +178,20 @@ func (m *Master) InstallAPIs(c *Config) {
 		apiGroupsInfo = append(apiGroupsInfo, apiGroupInfo)
 	}
 
+	// Run the tunneler.
```

Comment:
This is fixing an existing bug I noticed.

The tunneler uses the `m.getNodeAddresses` function, which calls `m.nodeRegistry.ListNodes()`. `m.nodeRegistry` is initialized in `m.initV1ResourcesStorage()`, which isn't called until after the tunneler starts trying to get node addresses. Right now, we panic and recover in a `util.Until` loop until the storage is initialized.